client/scripts/controllers/server/threads.ts

```diff
@@ -224,6 +249,83 @@ class ThreadsController {
     });
   }
 
+  public async fetchThread(id) {
+    const params = {
+      chain: app.activeChainId(),
+      community: app.activeCommunityId(),
+      id,
+    };
+    const response = await $.get(`${app.serverUrl()}/getThread`, params);
+    if (response.status !== 'Success') {
+      throw new Error(`Unsuccessful refresh status: ${response.status}`);
+    }
+    const thread = modelFromServer(response.result);
+    const existing = this._store.getByIdentifier(thread.id);
+    if (existing) this._store.remove(existing);
+    this._store.add(thread);
+    return modelFromServer(response.result);
+  }
+
+  // loadNextPage returns false if there are no more threads to load
+  public async loadNextPage(options: {
+    chainId: string,
+    communityId: string,
+    cutoffDate: moment.Moment,
```

Comment:
what is cutoff_date? sounds like the starting date (or the ending date?) of posts to retrieve

if it's not obvious, can we add a comment explaining so